client/js/mini-app/questions/resource-carousel.js

```diff
@@ -179,6 +178,7 @@ const ResourceCarousel = ({ classes, onResourceChange }) => (
 
 const styles = {
 	resourceCarouselRootPhone: {
+		'-webkit-tap-highlight-color': 'rgba(0, 0, 0, 0)',
```

Comment:
This change looks good!

Reply:
@vzajkov Ok I tested this on both an ipad and a phone. It's fixed on the phone, but not on the tablet. The reason is that this is setting the style at the root level. If you set it at the level where the grey box is showing up, it will fix for all devices (shown below).